api/middleware/validation.js

```diff
@@ -17,3 +18,21 @@ exports.validateAndSanitizeIncidentQueries = async (req, _res, next) => {
     next({ status: 400, message: err.message });
   }
 };
+
+exports.validateIncident = async (req, _res, next) => {
+  if (!req.body) {
+    return next({ status: 400, message: 'incident object required' });
+  }
+
+  try {
+    const validIncident = await incidentObjectSchema.validate(req.body, {
+      stripUnknown: true,
+    });
+
+    req.sanitizedIncident = validIncident;
+    delete req.body;
```

Comment:
Why delete the request body? Just curious

Reply:
Great question! It's no longer going to be used and as a dirty user input better safe than sorry.